pkgs/top-level/emacs-packages.nix

```diff
@@ -1843,5 +1834,6 @@ in
     // melpaPackages self
     // elpaPackages self
     // melpaStablePackages self
+    // orgPackages self
```

Comment:
Given its specificity---it's only got the two org packages in it---giving it precedence over everything else seemed right at the time, but I could be convinced otherwise.